### PR TITLE
Meet a Tesselator, Nick Hehr

### PR DESCRIPTION
**Pull request by HipsterBrown**
Related to #24 

Pinging @Frijol for review and feedback.

### CODE REVIEW

File: 2016-06-23.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+
+### What is something you enjoyed building with Tessel?
+One of my early projects was something I built for a local meetup talk, showing how I got started with Tessel as someone new to hardware. With a plain  t-shirt, an Adafruit Gemma, some sewable LEDs, and a Tessel 2, I served a web app through a Tessel access point that allowed folks in the audience to control the lights on my shirt. It was really fun to create an interactive hardware demo that didn't depend on the event's wifi network to work. The most difficult part for the whole project ended up being the sewing! All the detail for building that project can be found in [this repo](https://github.com/hipsterbrown/tessel-router/tree/gemma-control).
```
Comment (Frijol) [inline 2016-06-23.md]: I love this! I didn't know you'd done that!

**Comment by Frijol** [comment]
A++, nothing to add!

**Comment by Frijol** [comment]
(do you know how to deploy it to the blog?)

**Comment by Frijol** [comment]
oh– also, should add something to the file name. The date is good, but we should start using more descriptive titles now that we're branching out from just TWIT. 2016-06-23-Nick-Hehr.md?

**Comment by HipsterBrown** [comment]
I don't know how to deploy this but that could be a nice update to the README if possible. 

I'll push an update to file name.

**Comment by Frijol** [comment]
ready to go?

**Comment by HipsterBrown** [comment]
@Frijol Yup, I think so! :smile:

**Pull request merged**